server.js

```diff
@@ -27,9 +19,59 @@ const app = express()
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+//Model for Book
+const Book = new mongoose.model('Book', {
+  bookID: Number,
+  title: String,
+  authors: String,
+  average_rating: Number,
+  language_code: String,
+  num_pages: Number,
+  ratings_count: Number,
+  text_reviews_count: Number
+})
```

Comment:
Great choice to use this setup for this instead of writing the code like: 
text_reviews_count: {
     type: Number,
   },